politica-w-ibid.csl

```diff
@@ -0,0 +1,351 @@
+    <updated>2025-08-31T15:00:00+01:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="da-DK">
+    <terms>
+      <term name="editor" form="short">red.</term>
+      <term name="editor">red.</term>
+      <term name="et-al">et al.</term>
+      <term name="and">og</term>
+      <term name="page" form="short">s.</term>
+      <term name="page">s.</term>
+      <term name="month-01">januar</term>
+      <term name="month-02">februar</term>
+      <term name="month-03">marts</term>
+      <term name="month-04">april</term>
+      <term name="month-05">maj</term>
+      <term name="month-06">juni</term>
+      <term name="month-07">juli</term>
+      <term name="month-08">august</term>
+      <term name="month-09">september</term>
+      <term name="month-10">oktober</term>
+      <term name="month-11">november</term>
+      <term name="month-12">december</term>
+      <term name="accessed">hentet</term>
+      <term name="ibid">ibid.</term>
+    </terms>
+  </locale>
+  <macro name="editor">
+    <names variable="editor">
+      <name and="text" delimiter=", " delimiter-precedes-last="never"/>
+      <label form="short" text-case="lowercase" prefix=" (" suffix=")" strip-periods="true"/>
+    </names>
+  </macro>
+  <macro name="anon">
+    <text term="anonymous" form="short" text-case="capitalize-first" strip-periods="true"/>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name and="text" delimiter-precedes-last="never" name-as-sort-order="first"/>
+      <label form="short" prefix=", " text-case="lowercase"/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+        <text macro="anon"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" and="text" delimiter-precedes-last="never" et-al-min="4"/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+        <text macro="anon"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="access">
+    <choose>
+      <if type="article article-journal book chapter thesis" match="none">
+        <choose>
+          <if variable="URL">
+            <text variable="URL"/>
+            <group prefix=" (" suffix=")">
+              <text term="accessed" text-case="capitalize-first" suffix=" "/>
+              <date variable="accessed">
+                <date-part name="day" suffix=". "/>
+                <date-part name="month" text-case="lowercase" suffix=" "/>
+                <date-part name="year"/>
+              </date>
+            </group>
+          </if>
+        </choose>
+      </if>
+    </choose>
+  </macro>
+  <macro name="title">
+    <choose>
+      <if type="thesis">
+        <text variable="title"/>
```

Comment:
if none of the titles are styled differently, this whole conditional is not necessary.